s/shiro/shiro_1.5.2_ubi_8.4.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : shiro
+# Version               : shiro-root-1.5.2
```

Comment:
is the script very specific to this version? If not, maybe we can remove the version from the script name?

Reply:
Renamed as suggested.